### PR TITLE
Readme.md Tweak Command Line section

### PR DESCRIPTION
**Pull request by Craftyawesome**
Adds newer backends to readme + more positive about hle accuracy.

The command list itelf is still outdated, though.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -141,12 +141,13 @@ see where it's stored) if you don't plan to reinstall Dolphin.
 * -A, --audio_emulation=<str> Low level (LLE) or high level (HLE) audio
 
 Available DSP emulation engines are HLE (High Level Emulation) and
-LLE (Low Level Emulation). HLE is fast but often less accurate while LLE is
+LLE (Low Level Emulation). HLE is fast but can be less accurate while LLE is
```
Comment (jordan-woyak) [inline Readme.md]: I think "often" is still a better word here.

Comment (Craftyawesome) [reply Readme.md]: Is HLE incorrect enough to be "often" less accurate than LLE, though?
Maybe something like
```suggestion
LLE (Low Level Emulation). HLE is fast but is less accurate than LLE, which is
```

File: Readme.md [hunk 2]
```diff
@@ -141,12 +141,13 @@ see where it's stored) if you don't plan to reinstall Dolphin.
...
 
 Available DSP emulation engines are HLE (High Level Emulation) and
-LLE (Low Level Emulation). HLE is fast but often less accurate while LLE is
+LLE (Low Level Emulation). HLE is fast but is less accurate than LLE, which is
 slow but close to perfect. Note that LLE has two submodes (Interpreter and
 Recompiler), which cannot be selected from the command line.
 
-Available video backends are "D3D" (only available on Windows) and
-"OGL". There's also "Software Renderer", which uses the CPU for rendering and
+Available video backends are "D3D", "D3D12", (only available on Windows) "OGL", and "Vulkan".
```
Comment (leoetlino) [inline Readme.md]: One comma is misplaced:
 
```suggestion
Available video backends are "D3D", "D3D12" (only available on Windows), "OGL", and "Vulkan".
```

**Comment by Rukario** [comment]
It'd probably read better with less use of the word "which",

"HLE is faster but less accurate whereas LLE is slower but close to perfect. Note that LLE has two submodes (Interpreter and Recompiler) but they cannot be selected from the command line."

And then

"Available video backends are "D3D" and "D3D12" (they are only available on Windows), "OGL", and "Vulkan"."

It reads better with an "and" instead of comma for the D3D modes.

**Comment by leoetlino** [comment]
That does read better. @Craftyawesome would you mind making those changes?

**Pull request merged**